query/organization/org_default_repo_permission_none_read.sql

```diff
@@ -2,11 +2,16 @@ select
   -- Required Columns
   html_url as resource,
   case
+    when default_repo_permission is null then 'skip'
```

Comment:
Please see comments for the other query, I believe they should mostly be the same here as well.